libraries/chain/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-file(GLOB HEADERS "include/koinos/chain/*.hpp" "include/koinos/chain/wasm/*.hpp")
+file(GLOB HEADERS "include/koinos/chain/*.hpp")
 add_library(koinos_chain_lib
             apply_context.cpp
             controller.cpp
-            host.cpp
+            koinos_api_handler.cpp
```

Comment:
`api_handler` is the upward-facing interface of `vmmanager`.  If you're a user of `vmmanager` and you want to tell `vmmanager` how to handle function calls, you do it by implementing an `api_handler`.

`host_api` is the downward-facing interface of `vmmanager`, and it is a specific detail of the EOS-VM backend.  If you're `vmmanager` and you want to tell EOS-VM how to handle function calls, you do it by implementing `host_api`.  Fizzy doesn't have an equivalent of `host_api`, you tell it how to service functions in the C style (giving a function pointer to a callback that takes a user-provided `void *` parameter to pass data into the callback).

They have different names because they fit in different places architecturally.  We can't change the names the EOS VM uses without editing the EOS VM source, so I changed the Koinos names.